frameworks/hdfs/universe/resource.json

```diff
@@ -8,7 +8,8 @@
       "hdfs-jre-tar-gz": "{{jre-jce-unlimited-url}}",
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
       "scheduler-zip": "{{artifact-dir}}/hdfs-scheduler.zip",
-      "executor-zip": "{{artifact-dir}}/executor.zip"
+      "executor-zip": "{{artifact-dir}}/executor.zip",
+      "zone-resolver-sh": "http://downloads.mesosphere.com/hdfs/assets/zone-resolver.sh"
```

Comment:
Why not template this?

Reply:
Also, this script needs to be version controlled in this repo

Reply:
We _could_ put it as a template/config file, though there's not much to template in it.

Reply:
I meant, "why not abuse templating to copy over the simple script?"

Having it as a URI is fine of course.